cli/src/cluster_query.rs

```diff
@@ -1196,9 +1213,12 @@ pub fn process_get_epoch_info(rpc_client: &RpcClient, config: &CliConfig) -> Pro
                     .get_block_time(first_block_in_epoch)
                     .ok()
                     .map(|time| {
-                        time - (((first_block_in_epoch - epoch_expected_start_slot)
-                            * average_slot_time_ms)
-                            / 1000) as i64
+                        time.saturating_sub(
+                            first_block_in_epoch
+                                .saturating_sub(epoch_expected_start_slot)
+                                .saturating_mul(average_slot_time_ms)
+                                .saturating_div(1000) as i64,
```

Comment:
I will admit this is much nicer to read

Reply:
Partially it is just luck :)
`fmt` does a suboptimal job for some complex mathematical expressions.
Maybe something like this would be a bit easier to follow:

```rust
  time - (
    (
      (first_block_in_epoch - epoch_expected_start_slot) * average_slot_time_ms
    ) / 1000
  ) as i64
```
It would be better to introduce intermediate variables, rather than writing such a complex expression as one thing.
Intermediates would provide additional names, clarifying the computation further.

Having said that, I take the win, if you think I'm doing a good job :))

Reply:
Of course!